core/src/com/unciv/ui/worldscreen/mainmenu/DropBox.kt

```diff
@@ -126,6 +126,16 @@ class OnlineMultiplayer {
         return GameSaver.gameInfoFromString(Gzip.unzip(zippedGameInfo))
     }
 
+    /**
+     * WARNING!
+     * Does not initialize transitive GameInfo data.
+     * It is therefore stateless and save to call for Multiplayer Turn Notifier, unlike tryDownloadGame().
+     */
+    fun tryDownloadGameUninitialized(gameId: String): GameInfo {
```

Comment:
Looks to me like this and tryDownloadCurrentTurnCiv are almost the same, I thing you could have tryDownloadCurrentTurnCiv just call this function and take the civ out of it, removing the GameSaver.currentTurnCivFromString function entirely

Reply:
Actually looking at it again it seems that GameSaver.currentTurnCivFromString was only used from the turnChecker file anyway, and so now that you've removed that reference it should just be completely deleted (along with the accompanying GameSaver.currentTurnCivFromString function)

Reply:
Removed both tryDownloadCurrentTurnCiv and currentTurnCivFromString with last commit